darwin/exporter/formats/nifti.py

```diff
@@ -161,10 +234,13 @@ def get_view_idx(frame_idx, groups):
             return view_idx
 
 
-def get_view_idx_from_slot_name(slot_name):
-    slot_names = {"0.1": 0, "0.2": 1, "0.3": 2}
-    slot_names.get(slot_name, 0)
-    return slot_names.get(slot_name, 0)
+def get_view_idx_from_slot_name(slot_name, orientation):
+    if orientation is None:
+        orientation_dict = {"0.1": 0, "0.2": 1, "0.3": 2}
+        return orientation_dict.get(slot_name, 0)
+    else:
+        orientation_dict = {"AXIAL": 0, "SAGITTAL": 1, "CORONAL": 2}
```

Comment:
Could be an enum, no change necessary

Reply:
Which part are you referring to for an enum?

Reply:
The orientation dictionaries
```python
from enum import Enum

class Orientation(Enum):
    AXIAL=0
    SAGITTAL=1
    CORONAL=2
    
orientation = Orientation.AXIAL
```

Reply:
can also specify defaults for missing values for the more complex slotted dict on L239
```python
class Orientation(Enum):
    AXIAL=0
    SAGITTAL=1
    CORONAL=2
    @classmethod
        def _missing_(cls, value):
            return cls.AXIAL
            
```